airbyte-integrations/bases/base-normalization/build.gradle

```diff
@@ -10,30 +10,48 @@ airbytePython {
 }
 
 dependencies {
-    implementation project(':airbyte-connector-test-harnesses:acceptance-test-harness')
+    project(':airbyte-connector-test-harnesses:acceptance-test-harness')
```

Comment:
This is because we no longer apply the `java` plugin blindly everywhere. See `build.gradle`.